examples/guided.py

```diff
@@ -344,7 +344,11 @@ def perform_installation(device, boot_partition, language, mirrors):
 
 			if (root_pw := archinstall.arguments.get('!root-password', None)) and len(root_pw):
 				installation.user_set_pw('root', root_pw)
-
+			if archinstall.arguments.get('profile', None) == "i3-wm" or archinstall.arguments.get('profile', None) == "i3-gaps":
+				print("the installation of i3/i3-gaps does not conatain any configuerations for the wm. in this shell you should add your configuerations")
+				subprocess.check_call("arch-chroot /mnt")
```

Comment:
Just a note here, I think we should design a `_post_instruction()` function for profiles, much like we have for `_prep_function()`. As this might grow out of control if other profiles start adding the same code to guided. This way guided can support multiple profiles with the same need - from one block of code rather than having to check `if profile == "someProfile"`.

Perhaps:
```python
if profile.has_post_function():
    profile._post_function()
```
And then we copy and adopt `Profile().has_prep_function()` into a new one called `Profile().has_post_function()`.

Reply:
Yeah I'll add that in. Its a bit late for me so I'll do it in the morning

Reply:
I'll merge this for now and you can add that as a new feature in the morning. As it will add new functionality it's nice to have a separate PR for it.